.github/workflows/build.yml

```
@@ -114,28 +165,29 @@ jobs:
       with:
         arch: amd64
     - name: Add LLVM to PATH
-      if: ${{ matrix.compiler == 'clang-cl' }}
       run: echo "${env:LLVM_PATH}" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
     - name: Source checkout
       uses: actions/checkout@v2
       with:
         path: src
     - name: Configure
-      run: cmake -S src -B build -G Ninja -DCMAKE_TOOLCHAIN_FILE:FILEPATH=toolchain-msvc.cmake -DARCH:STRING=${{matrix.arch}} -DENABLE_ROSTESTS=1 -DENABLE_ROSAPPS=1 -DUSE_CLANG_CL:BOOL=${{matrix.compiler == 'clang-cl'}}
+      run: cmake -S src -B build -G Ninja -DCMAKE_TOOLCHAIN_FILE:FILEPATH=toolchain-msvc.cmake -DARCH:STRING=${{matrix.arch}} -DENABLE_ROSTESTS=1 -DENABLE_ROSAPPS=1 -DUSE_CLANG_CL:BOOL=TRUE
     - name: Build
       run: cmake --build ${{github.workspace}}\build
     - name: Generate ISOs
       run:  cmake --build ${{github.workspace}}\build --target bootcd --target livecd
-    - name: Upload bootcd
+    - name: Upload ISOs & symbols
       uses: actions/upload-artifact@v2
       with:
-        name: reactos-bootcd-${{matrix.compiler}}-${{matrix.arch}}-${{github.sha}}
-        path: build/bootcd.iso
-    - name: Upload livecd
+        name: reactos-clang-cl-${{matrix.arch}}-${{github.sha}}
```

```suggestion
        name: reactos-isos-clang-cl-${{matrix.arch}}-${{github.sha}}
```
And the other two cases too.

---

The names are already long enough as it is, and the other artifacts have the syms part differentiating them.